.drone.star

```diff
@@ -742,7 +742,7 @@ def phpTests(ctx, testType, withCoverage):
 
         for phpVersion in params["phpVersions"]:
             if testType == "phpunit":
-                if params["coverage"]:
+                if params["coverage"] and phpVersion not "7.3":
```

Comment:
This is in common code that is exactly the same in all oC10 apps. I really don't want to change that - unless we decide to do implement this "rule" for every oC10 app. And that might be a bit confusing when someone explicitly sets "coverage" true and PHP 7.3 in the config section, but does not get any coverage.

Reply:
Ahh okay. How do you suggest to skip the coverage then?